[Bug] [Alert] batchInsert execption2

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

https://github.com/apache/dolphinscheduler/issues/14675 https://github.com/apache/dolphinscheduler/issues/14676

After resolving the previous issue, there is still a lingering problem.

When I configure an alarm instance for the alarm group, everything works fine. However, when I configure two alarm instances for the alarm group, there is an issue with bulk insertion.

The exception in alert-server is as follows:

```
[ERROR] 2023-08-01 19:55:05.286 +0800 org.apache.dolphinscheduler.alert.service.AlertBootstrapService:[90] - Alert sender thread meet an exception
org.springframework.jdbc.BadSqlGrammarException: 
### Error updating database.  Cause: java.sql.SQLException: Column count doesn't match value count at row 1
### The error may exist in class path resource [org/apache/dolphinscheduler/dao/mapper/AlertSendStatusMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: insert into t_ds_alert_send_status (alert_id, alert_plugin_instance_id, send_status, log, create_time)         values          (               ?,             ?,             ?,             ?,             ?          ,              ?,             ?,             ?,             ?,             ?          )
### Cause: java.sql.SQLException: Column count doesn't match value count at row 1
; bad SQL grammar []; nested exception is java.sql.SQLException: Column count doesn't match value count at row 1
```


https://github.com/apache/dolphinscheduler/blob/139f23040a0abc14a23872d4426196d558044c61/dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/AlertSendStatusMapper.xml#L21-L31

### What you expected to happen

no error

### How to reproduce

as mentioned above

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/AlertSendStatusMapper.xml
+++ b/dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/AlertSendStatusMapper.xml
@@ -22,14 +22,13 @@
     <insert id="batchInsert">
         insert into t_ds_alert_send_status (alert_id, alert_plugin_instance_id, send_status, log, create_time)
         values
-        <foreach collection="alertSendStatuses" item="alertSendStatus" separator="," open="(" close=")">
-            #{alertSendStatus.alertId},
+        <foreach collection="alertSendStatuses" item="alertSendStatus" separator=",">
+            (#{alertSendStatus.alertId},
             #{alertSendStatus.alertPluginInstanceId},
             #{alertSendStatus.sendStatus},
             #{alertSendStatus.log},
-            #{alertSendStatus.createTime}
+            #{alertSendStatus.createTime})
         </foreach>
-
     </insert>
 
     <delete id="deleteByAlertIds">

